TomatoBar/TomatoBarController.swift

```diff
@@ -69,8 +81,10 @@ public class TomatoBarController: NSViewController {
     override public func viewDidLoad() {
         super.viewDidLoad()
         /* Register defaults */
-        UserDefaults.standard.register(defaults: ["intervalLength": 25,
-                                                  "isSoundEnabled": true])
+        UserDefaults.standard.register(defaults: ["workingIntervalLength": 25,
+                                                  "isSoundEnabled": true,
+                                                  "restingIntervalLength": 5,
+                                                  "isResting": true])
```

Comment:
isResting is an app state variable, not user setting, it shouldn't be registered here

Reply:
thanks for the feedback, i'm really new to swift/macos development and i don't know how to do somethings